OpenRA.Game/Settings.cs

```diff
@@ -101,6 +101,7 @@ public ServerSettings(ServerSettings other)
 	public class DebugSettings
 	{
 		public bool BotDebug = false;
+		public bool LUADebug = false;
```

Comment:
Lua, not LUA.  (http://www.lua.org/about.html even has a special section about this).